Gemfile

```diff
@@ -43,7 +43,6 @@ mingw << :x64_mingw if Bundler::Dsl::VALID_PLATFORMS.include?(:x64_mingw)
 platform(*mingw) do
   gem 'ffi', '~> 1.9.5', :require => false
   gem 'win32-dir', '~> 0.4.8', :require => false
-  gem 'win32-security', '~> 0.2.5', :require => false
```

Comment:
Ok, I'll add the dependency back in and a comment saying that use of the gem is deprecated, as I did in the Puppet repo.